### PR TITLE
Add missing apt-get update to have an up-to-date package list

### PR DESCRIPTION
**Pull request by keryell**
Also remove random TAB and spaces.
This file would require a serious beautifier.

### CODE REVIEW

File: src/runtime_src/tools/scripts/xrtdeps.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -vx
```
Comment (stsoe) [inline src/runtime_src/tools/scripts/xrtdeps.sh:1]: Really?

**Review by stsoe — CHANGES_REQUESTED** [review]

**Comment by stsoe** [comment]
@keryell Please `git commit -s --amend --no-edit`

File: src/runtime_src/tools/scripts/xrtdeps.sh (hunk 1 above)
Comment (keryell) [reply src/runtime_src/tools/scripts/xrtdeps.sh:1]: Ouch, I pushed my debug code. :-(

**Comment by stsoe** [comment]
> Actually in Emacs/magit I just need to press 'r' and 'm'. I tend to forget about all the git CLI...

I was implying you need to sign your PR.

**Comment by keryell** [comment]
> > @keryell Please `git commit -s --amend --no-edit`
> 
> Actually in Emacs/magit I just need to press 'r' and 'm'. I tend to forget about all the git CLI...

Oh, this is a new policy? By signing I guess this is just adding my e-mail, no cryptographic signature?

**Comment by stsoe** [comment]
ok to test

**Comment by stsoe** [comment]
add to whitelist

**Pull request merged**